lms/envs/common.py

```diff
@@ -2372,6 +2372,7 @@
 FINANCIAL_REPORTS = {
     'STORAGE_TYPE': 'localfs',
     'BUCKET': 'edx-financial-reports',
+    'CUSTOM_DOMAIN': 'edx-financial-reports.s3.amazonaws.com',
```

Comment:
If `CUSTOM_DOMAIN` is an S3/Boto convention then let's stick with it -- thanks for the info.